### PR TITLE
Added support for --one-file-system for Borg.

### PR DESCRIPTION
**Pull request by ypid**
Note that I already resolved the merge conflicts with #5 in https://github.com/ypid/atticmatic/tree/merge in case you decide to merge both PR.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -2,3 +2,4 @@ Dan Helfman <witten@torsion.org>: Main developer
 
 Alexander Görtz: Python 3 compatibility
 Henning Schroeder: Copy editing
+Robin `ypid` Schneider: Support additional options of Borg
```
Comment (witten) [inline AUTHORS]: Ah, great. I see you found the AUTHORS file in this changeset. :)

File: sample/config [hunk 2]
```diff
@@ -2,6 +2,8 @@
 # Space-separated list of source directories to backup.
 source_directories: /home /etc
 
+# one_file_system: 1
```
Comment (witten) [inline sample/config]: Maybe add a brief comment above this line describing it?

File: atticmatic/backends/borg.py [hunk 3]
```diff
@@ -7,7 +7,14 @@
 
 COMMAND = 'borg'
 CONFIG_FORMAT = (
-    shared.CONFIG_FORMAT[0],  # location
+    Section_format(
+        'location',
+        (
+            option('source_directories'),
+            option('one_file_system', required=False),
```
Comment (witten) [inline atticmatic/backends/borg.py]: It's probably a good idea here to pass in `value_type=bool`. Note that if you do that, you'll have to alter `type_getter` in `parse_section_options()`, adding something like `bool: parser.getboolean,`

File: sample/config (hunk 2 above)
Comment (witten) [inline sample/config]: This example might be clearer if the value was `true`. See: https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getboolean

File: atticmatic/tests/unit/backends/test_shared.py [hunk 4]
```diff
@@ -131,6 +131,22 @@ def test_create_archive_with_compression_should_call_attic_with_compression_para
     )
 
 
+def test_create_archive_with_one_file_system_should_call_attic_with_one_file_system_parameters():
```
Comment (witten) [inline atticmatic/tests/unit/backends/test_shared.py]: Yay tests.

**Comment by witten** [comment]
Please add this great new feature to the NEWS file!

File: atticmatic/tests/unit/backends/test_shared.py (hunk 4 above)
Comment (ypid) [inline atticmatic/tests/unit/backends/test_shared.py]: I am just continuing your great work :+1: :wink:

**Comment by ypid** [comment]
You are right. Thanks for the review. Done.

**Comment by witten** [comment]
Manually merged this in 049f9c8853ef287a547e55ab2b3d02ea6b0c6cd7. My only change was altering the version number to make Python's version normalization happy.

Thanks!

**Comment by ypid** [comment]
Thanks. Just a note. Git supports author and committer separately.

**Comment by witten** [comment]
I'll look into that for future merges. I'm using [HgGit](https://www.mercurial-scm.org/wiki/HgGit) as a bridge from Mercurial to GitHub, so it may or may not support that. I suppose I can always use git directly for such merges and then pull them back to my Mercurial repo.

**Pull request closed without merging**